docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,259 @@
+    misleading, especially for imbalanced data.
+*   `Precision <https://en.wikipedia.org/wiki/Precision_and_recall>`__ - how many samples were classified correctly as
+    positive out of the total number of samples classified as positive. Can be also viewed as the fraction of relevant
+    samples out of the samples spotted by the model.
+*   `Recall <https://en.wikipedia.org/wiki/Precision_and_recall>`__ /
+    `Sensitivity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__ / TPR - how many samples were classified
+    correctly as positive out of the total number of positive samples. Can be also viewed as the fraction of the
+    relevant samples that were spotted by the model.
+
+
+.. image:: /_static/images/general/PrecisionRecall_wikipedia.svg.png
+   :alt: Precision and recall
+   :align: center
+
+Precision and Recall by Walber - Own work, CC BY-SA 4.0,
+`source <https://commons.wikimedia.org/w/index.php?curid=36926283>`__
+
+
+*   `Specificity <https://en.wikipedia.org/wiki/Sensitivity_and_specificity>`__/ TNR - how many samples were classified
+    correctly as negative out of the total number of negative samples. Complementary to sensitivity.
+
+.. image:: /_static/images/general/Sensitivity_and_specificity_wikipedia.svg.png
+   :alt: Specificity and sensitivity
+   :align: center
+
+Specificity and Sensitivity by FeanDoe - Modified version from Walber&#039;s Precision and Recall
+`source <https://commons.wikimedia.org/w/index.php?curid=94134880>`__
+
+*   `F-1 <https://en.wikipedia.org/wiki/F-score>`__ - combines the precision and recall into one metric by taking their
+    harmonic mean. More robust to class imbalance than accuracy.
+*   `AUC <https://developers.google.com/machine-learning/crash-course/classification/roc-and-auc>`__  - The area under
+    the curve of the ROC graph. For the previous classification metrics, a threshold on the model output was chosen
+    above which the sample is classified as positive and below as negative. The AUC takes into account all possible
+    thresholds.
+
+Object Detection
+----------------
+*   `Jaccard Index <https://en.wikipedia.org/wiki/Jaccard_index>`__ / IoU - the ratio between the overlapping area of
+    predicted and the actual bounding box and the union of their areas. The most intuitive metric for object detection.
+*   mAP - mean average precision. The mean of the average precision per class over the classes. Calculated at a certain
+    threshold, usually 0.5. Commonly used for benchmarking object detection models. A good explanation of how it is
+    calculated can be found
+    `here <https://towardsdatascience.com/breaking-down-mean-average-precision-map-ae462f623a52>`__.
+*   mAR - mean average recall. The mean of the average precision per class over the classes. Summed over the threshold
+    range [0.5, 1].
+
+How can I use metrics with Deepchecks?
```

Comment:
Would extend this section, mention all metrics currently supported by deepchecks and their deepchecks str name (for sklearn methods it is as mentioned here https://scikit-learn.org/stable/modules/model_evaluation.html)
Not sure the default are consistent across all check so maybe avoid saying what are the defaults
Would add example how to supply a metric to a specific check (as a list of strings to the alternative_scorers argument)

Reply:
@Nadav-Barak do we have this kind of example somewhere in the docs? This sklearn link is already there under "Alternative Metrics" (line 225)

Reply:
I think it should be an integral part here, we should describe all the metrics we support and their str value the user need to pass in our check to get those metrics
It is a big requirement, so better if you approve @shir22

Reply:
@Nadav-Barak, agreed to wait until the implementation of str and Sklearn metrics for vision is complete

Reply:
@TheSolY let's make sure there is an issue up to update this accordingly?

Reply:
@shir22 done https://github.com/deepchecks/deepchecks/issues/1705